package.json

```diff
@@ -6,8 +6,8 @@
         "dev": "run-s db:dev:migrate next:dev",
         "build": "run-s db:prod:migrate next:build",
         "start": "next start",
-        "next:build": "next build",
-        "next:dev": "next dev",
+        "next:build": "npm run chakra-types && next build",
```

Comment:
Running the `chakra-types` npm script is a requirement for the next build to succeed. I added it to the dev script also so that no ts errors, related to custom variants, are produced when you start your local dev environment.